speech-to-text/src/main/java/com/ibm/watson/developer_cloud/speech_to_text/v1/model/RecognizeOptions.java

```diff
@@ -140,7 +140,6 @@
     private Boolean profanityFilter;
     private Boolean smartFormatting;
     private Boolean speakerLabels;
-    private Boolean interimResults;
```

Comment:
Was removing of `intermResults` generated? I thought this was still supported by the service

Reply:
It's actually generated without, although that's a good catch because I have it written to add it manually in my post-gen notes.